src/main/flight/mixer.c

```diff
@@ -816,8 +821,14 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
 
     pidUpdateAntiGravityThrottleFilter(throttle);
 #ifdef USE_DYN_LPF
-    dynLpfGyroUpdate(throttle);
-    dynLpfDTermUpdate(throttle);
+    const int quantizedThrottle = lrintf(throttle * DYN_LPF_THROTTLE_STEPS); // quantize the throttle reduce the number of filter updates
+    if (quantizedThrottle != dynLpfPreviousQuantizedThrottle) {
+        // scale the quantized value back to the throttle range so the filter cutoff steps are repeatable
+        const float dynLpfThrottle = (float)quantizedThrottle / DYN_LPF_THROTTLE_STEPS;
+        dynLpfGyroUpdate(dynLpfThrottle);
```

Comment:
No need to introduce loss of precision here, just do it like this:
```cpp
dynLpfGyroUpdate(throttle);
dynLpfDTermUpdate(throttle);
```

getting rid of `dynLpfThrottle` altogether.

Reply:
@DieHertz : This will actually increase error - throttle will be latched when entering given interval.
It will be indeterministic too - latched throttle position is different every time

Reply:
That's variability and I don't see how it's a bad thing. Just look at at how the same approach is used e.g. for grid-based motion planning, where each cell (bucket) is associated with continuous state, and a state mapped to a given cell can assume any values in given range, not just the edge values.

Reply:
PR code uses rounding - center of interval is used..